python/inline/conftest.py

```diff
@@ -526,6 +564,17 @@ def parse_constructor(self, node):
                         and isinstance(keyword.value.value, bool)
                     ):
                         self.cur_inline_test.disabled = keyword.value.value
+                    # check if "timeout" is a positive integer
+                    elif (
+                        keyword.arg == self.arg_timeout_str
+                        and isinstance(keyword.value, ast.Num)
+                        and isinstance(keyword.value.n, int)
```

Comment:
It is better to use type "float" than "int" for timeout value

Reply:
Fixed